CMakeLists.txt

```diff
@@ -56,25 +56,29 @@ set(RUSTFLAGS $ENV{RUSTFLAGS})
 add_custom_target(cargo ALL COMMAND cargo build ${CARGO_FLAGS} COMMENT "Running RUSTFLAGS=${RUSTFLAGS} cargo build ${CARGO_FLAGS}")
 
 if(APPLE)
-	set(libzenohc "${CMAKE_BINARY_DIR}/libzenohc.dylib")
-	set(libzenohc_static "${CMAKE_BINARY_DIR}/libzenohc.a")
+	set(libzenohc "libzenohc.dylib")
+	set(libzenohc_static "libzenohc.a")
 elseif(UNIX)
-	set(libzenohc "${CMAKE_BINARY_DIR}/libzenohc.so")
-	set(libzenohc_static "${CMAKE_BINARY_DIR}/libzenohc.a")
+	set(libzenohc "libzenohc.so")
+	set(libzenohc_static "libzenohc.a")
 elseif(WIN32)
-	set(libzenohc "${CMAKE_BINARY_DIR}/zenohc.dll")
-	set(libzenohc_static "${CMAKE_BINARY_DIR}/zenohc.lib")
+	set(libzenohc "zenohc.dll")
+	set(libzenohc_static "zenohc.lib")
 endif()
```

Comment:
Any specific reason to have `${CMAKE_BINARY_DIR}` removed here and made it explicitly everywhere else?
It seems to me that it can be more prone to errors.

Reply:
Only for the PackageConfig, which needs the name without the binary_dir because it should refer to the install location. We could do variables `libzenohc_name` and `libzenohc_binary_path` or something to avoid the `${CMAKE_BINARY_DIR}` everywhere, let me know what you prefer.

Reply:
Ok. Then, let's keep it then as originally provided by you.